### DISCUSSION TITLE
Xiisad - Chrome API & Extension Manifest | Saturday, 16 November 2024, 8:00 PM

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#26] [Zoom Links]
Xiisada caawa waa muhiim wax badan ayaan ku falanqeeneynaa marka hala wada imaado Insha'Allah!

**zoom-link:**
https://us06web.zoom.us/j/88104187837?pwd=9DYJ7bJlhEKLvdmhb9PNxa0PkvnjfZ.1

**Comment by sharafdin** [comment]
10 Minutes ayaa hartay dhalinyaro.

### mentions:
@dugsiiyeinc/dhaqsopay-extension-developers 
@dugsiiyeinc/dhaqsopay-server-developers 
@dugsiiyeinc/dhaqsopay-website-developers

**Comment by A-M-Ahmed** [comment]
cawa cool  🔥

**Comment by ducesane** [comment]
Xalay cashir fiiican aa qadanay thanks @sharafdin

**Reply by sharafdin** [reply]
Adaa mudan akhii ❤️🙌🏿

**Comment by faaraxcabdulaahi** [comment]
Eng @sharafdin bro **si fiican baad noo dhiirigelisay** :+1:, xalay **2saac** baad waxaad nagu fahansiisay wax **2 bilood** nagu qaadan lahaa baaristeeda.
